File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 3 - C/q10.c
//A confederação brasileira de natação irá promover eliminatórias para o próximo mundial. Faça um
//algoritmo que receba a idade de um nadador e imprima a sua categoria segundo a tabela a seguir:
//Categoria Idade
//Infantil A 5 – 7 anos
//Infantil B 8 – 10 anos
//Juvenil A 11 – 13 anos
//Juvenil B 14 – 17 anos
//Sênior Maiores de 18 anos

#include<stdio.h>
int main(){
	int idade;
	printf("Idade: ");
	scanf("%d", &idade);
	if(idade>=5 && idade<=7)
	    printf("INFANTIL A");
	else if(idade>=8 && idade<=10)
	    printf("INFANTIL B");
	else if(idade>=11 && idade<=13)
	    printf("JUVENIL A");
	else if(idade>=14 && idade<=17)
	    printf("JUVENIL B");
	else if(idade>=18)
	    printf("SENIOR");

}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 3 - Python/q10.py
#A confederação brasileira de natação irá promover eliminatórias para o próximo mundial. Faça um
#algoritmo que receba a idade de um nadador e imprima a sua categoria segundo a tabela a seguir:
#Categoria Idade
#Infantil A 5 – 7 anos
#Infantil B 8 – 10 anos
#Juvenil A 11 – 13 anos
#Juvenil B 14 – 17 anos
#Sênior Maiores de 18 anos

idade=int(input("Insira a idade do atleta: "))
if idade>=5 and idade<=7:
    print("INFANTIL A")
elif idade>=8 and idade<=10:
    print("INFANTIL B")
elif idade>=11 and idade<=13:
    print("JUVENIL A")
elif idade>=14 and idade<=17:
    print("JUVENIL B")
elif idade>=18:
    print("SENIOR")
else:
    print("SEM CATEGORIA")
